include/boost/histogram/utility/jeffreys_interval.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 Jay Gohil, Hans Dembinski
+//
+// Distributed under the Boost Software License, version 1.0.
+// (See accompanying file LICENSE_1_0.txt
+// or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_HISTOGRAM_UTILITY_JEFFREYS_INTERVAL_HPP
+#define BOOST_HISTOGRAM_UTILITY_JEFFREYS_INTERVAL_HPP
+
+#include <boost/histogram/utility/binomial_proportion_interval.hpp>
+#include <cmath>
+#include <utility>
+
+namespace boost {
+namespace histogram {
+namespace utility {
+
+template <class ValueType>
+class jeffreys_interval : public binomial_proportion_interval<ValueType> {
+  using base_t = binomial_proportion_interval<ValueType>;
+
+public:
+  using value_type = typename base_t::value_type;
+  using interval_type = typename base_t::interval_type;
+
+  explicit jeffreys_interval(deviation d = deviation{1.0}) noexcept
+      : z_{static_cast<double>(d)} {}
```

Comment:
Like for the Clopper-Pearson interval, you rather want to accept `confidence_level` here instead of `deviation`.